src/librustc_middle/mir/interpret/error.rs

```diff
@@ -450,6 +450,9 @@ pub enum UnsupportedOpInfo {
     ReadForeignStatic(DefId),
     /// Could not find MIR for a function.
     NoMirFor(DefId),
+
+    // The variants below are only reachable from CTFE/const prop, miri will never emit them.
```

Comment:
Miri can emit the pointer-byte errors currently. We need to adjust the engine to fix that (https://github.com/rust-lang/miri/issues/841). So for now I think this comment needs to be moved down by two variants.

Reply:
Also could you update the doc comment above the enum here, at least remove the part about the fall-back?